### ISSUE TITLE
Composite plots to circular layout

### ISSUE THREAD

**Issue by LuffyLuffy** [YuLab-SMU/ggtree#301]
Dear Prof Yu,
I see a wonderful example of the ggtree data in the 7.5 sector.
https://yulab-smu.github.io/treedata-book/chapter7.html#gheatmap-ggnewscale

**library(ggplot2)
library(ggtree)

set.seed(2019-10-31)
tr <- rtree(10)

d1 <- data.frame(
  # only some labels match
  label = c(tr$tip.label[sample(5, 5)], "A"),
  value = sample(1:6, 6))

d2 <- data.frame(
  label = rep(tr$tip.label, 5),
  category = rep(LETTERS[1:5], each=10),
  value = rnorm(50, 0, 3)) 
g <- ggtree(tr) + geom_tiplab(align=TRUE)
p1 <- ggplot(d1, aes(label, value)) + geom_col(aes(fill=label)) + 
  coord_flip() + theme_tree2() + theme(legend.position='none')
p2 <- ggplot(d2, aes(x=category, y=label)) + 
  geom_tile(aes(fill=value)) + scale_fill_viridis_c() + 
  theme_tree2() 
library(aplot)
p2 %>% insert_left(g) %>% insert_right(p1, width=.5)**

I wonder if it is possible to create a circular layout of this data

![image](https://user-images.githubusercontent.com/15357865/82548217-8f97b680-9b8d-11ea-99e8-3c4d45ea3b98.png)

so that ggtree can really reporduce a figure like bellow. 
![image](https://user-images.githubusercontent.com/15357865/82548289-af2edf00-9b8d-11ea-86c1-2f0b320ac991.png)

### Prerequisites

+ [ ] Have you read [Feedback](https://guangchuangyu.github.io/ggtree/#feedback) and follow the [guide](https://guangchuangyu.github.io/2016/07/how-to-bug-author/)?
	* [ ] make sure your are using the latest release version
	* [ ] read the [documents](https://yulab-smu.github.io/treedata-book/)
	* [ ] google your question/issue

### Describe you issue

* [ ] Make a reproducible example (*e.g.* [1](https://gist.github.com/talonsensei/e1fad082657054207f249ec98f0920eb))
* [ ] your code should contain comments to describe the problem (*e.g.* what expected and actually happened?)

### Ask in right place

* [ ] for bugs or feature requests, post here (github issue)
* [ ] for questions, please post to [google group](https://groups.google.com/forum/#!forum/bioc-ggtree)

**Comment by xiangpin** [comment]
Yes, You can reporduce the figure using [`ggtreeExtra`](https://github.com/YuLab-SMU/ggtreeExtra)

**Comment by LuffyLuffy** [comment]
> Yes, You can reporduce the figure using [`ggtreeExtra`](https://github.com/YuLab-SMU/ggtreeExtra)

Awesome! 
I try the code in ggtreeExtra,  the starshape seem not to work.

Error in starGrob(x = coords$x, y = coords$y, gp = gpar(fill = alpha(coords$fill,  : 
  the starshape should be one of 1 to 12 !
 Warning message:
Removed 229 rows containing missing values (geom_star).

**Comment by xiangpin** [comment]
The version of ggstar in README is development. You can update the vesion of [`ggstar`](https://github.com/xiangpin/ggstar/) from `github`, or you can use `scale_starshape_manual` to set `1~12` values.

**Comment by LuffyLuffy** [comment]
> The version of ggstar in README is development. You can update the vesion of [`ggstar`](https://github.com/xiangpin/ggstar/) from `github`, or you can use `scale_starshape_manual` to set `1~12` values.

Great! Problem solved out!
The ggstar is also great! 
The pch is limit in to several  groups. 
You guys are asesome!
